vNext/test/NonSiloTests/SerializationTests/ConfigurationTests/ClientConfigurationForSerializer.xml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ClientConfiguration xmlns="urn:orleans">
+  <Gateway Address="localhost" Port="40000"/>
+  <!-- To turn tracing off, set DefaultTraceLevel="Off" and have no overrides.
+    For the trace log file name, {0} is replaced by "Client" and {1} is the current time. -->
+  <Tracing DefaultTraceLevel="Info" TraceToConsole="false" TraceToFile="{0}-{1}.log" BulkMessageLimit="1000">
+  </Tracing>
+  <Statistics MetricsTableWriteInterval="300s" PerfCounterWriteInterval="30s" LogWriteInterval="300s" WriteLogStatisticsToTable="true" StatisticsCollectionLevel="Info"/>
+  <Messaging ResponseTimeout="30s" ClientSenderBuckets="8192" MaxResendCount="0">
+    <SerializationProviders>
+      <Provider type="UnitTests.Serialization.FakeSerializer, Orleans.NonSiloTests"/>
+    </SerializationProviders>
```

Comment:
Copied over the entire config file because FakeSerializer is in different assembly in vNext. Open to suggestion about 
1. link vs copy
2. port them to another project which share the same assembly name as main solution.